libs/pilight/core/common.c

```diff
@@ -111,7 +112,7 @@ void array_free(char ***array, int len) {
 	}
 }
 
-unsigned int explode(char *str, const char *delimiter, char ***output) {
+unsigned int explode(const char *str, const char *delimiter, char ***output) {
```

Comment:
I don't believe explode is limited to `const char`. You can bypass error by doing:
```
const char *a = 'foo';
char *p = a;
explode(a, ...);
```